project_reach/lib/message_filters.py

```diff
@@ -1,17 +1,22 @@
 from dateutil.parser import isoparse
 
 
-# TODO: Move to Core
+# TODO: Move to Core once adapted for and tested on a pipeline supports multiple radio shows
 class MessageFilters(object):
     # TODO: Log which data is being dropped?
     @staticmethod
     def filter_test_messages(messages, test_run_key="test_run"):
         return [td for td in messages if not td.get(test_run_key, False)]
 
+    @staticmethod
+    def filter_empty_messages(messages, message_key):
+        # TODO: Before using on future projects, consider whether messages which are "" should be considered as empty
+        return [td for td in messages if message_key in td]
```

Comment:
I had to read this code too many times. It could be 'td' being undescriptive, esp. without a type signature, or it could be that this is just hard to read as a comprehension due to too much repetition